app/views/portal/show.rb

```diff
@@ -507,20 +513,20 @@ def named_entity_field_label_i18n_key(field)
       map.key?(field) ? map[field] : field
     end
 
-    def agt_link_field?(field)
+    def named_entity_link_field?(field)
       [:about, :broader].include?(field)
     end
 
-    def normalise_named_entity(named_entity, agt_link = false)
+    def normalise_named_entity(named_entity, foldable_link = false)
```

Comment:
Cyclomatic complexity for normalise_named_entity is too high. [7/6]<br>Perceived complexity for normalise_named_entity is too high. [8/7]